Documentation/ExtensionArchitecture/Extbase/Reference/UriArguments.rst

```diff
@@ -0,0 +1,63 @@
+..  include:: /Includes.rst.txt
+
+..  index:: Extbase; URI arguments; arguments
+
+===================================
+URI arguments and reserved keywords
+===================================
+
+Extbase uses special URI arguments to pass variables
+to Controller arguments and the framework itself.
+
+..  todo: Check for completion of this
+
+Extbase uses a prefixed URI argument scheme that relies
+on plugin configuration.
+
+For example, the example extension `EXT:blog_example` would use:
+
+..  code-block:: plaintext
+
+    https://example.org/blog/?tx_blogexample_bloglist[action]=show&tx_blogexample_bloglist[controller]=Post&tx_blogexample_bloglist[post]=4711&cHash=...
```

Comment:
Can we break this into several lines? This causes horizontal scrolling

Reply:
Yes, good idea!